src/ui/digital-exchange/RatingFilterContainer.js

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import { convertToQueryString, FILTER_OPERATORS } from '@entando/utils';
+import { fetchDEComponents } from 'state/digital-exchange/components/actions';
+import RatingFilter from './RatingFilter';
```

Comment:
We use path relative to `src` directory, so this should be 
```import RatingFilterContainer from 'ui/digital-exchange/RatingFilterContainer';```

Reply:
fixed in commit 94e21ab0189f339d67a26b342b6df9905ab79739